### ISSUE TITLE
choco uninstall microsoft-office-deployment doesn’t work. 

### ISSUE THREAD

**Issue by bmcatherasoo** [open-circle-ltd/chocolatey.microsoft-office-deployment#32]
Comment from https://community.chocolatey.org/packages/microsoft-office-deployment 
http://disq.us/p/3075p6w

**Comment by FreeMinded** [comment]
@bmcatherasoo I don't think it ever worked. The Office Deployment Tool is just for deploying the desired Office version. It does not update or uninstall it.
If you have a solution to add a working uninstall script for all the possible Office variants we happily take PRs. :blush:
